src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -100,15 +110,22 @@ public void guiActionSelectDocument(boolean autoSelectForSingle) {
         } catch (DisposedException ex) {
             OOError.from(ex).setTitle(errorTitle).showErrorDialog(dialogService);
         } catch (WrappedTargetException
-                | IndexOutOfBoundsException
-                | NoSuchElementException ex) {
+                 | IndexOutOfBoundsException
+                 | NoSuchElementException ex) {
             LOGGER.warn("Problem connecting", ex);
             OOError.fromMisc(ex).setTitle(errorTitle).showErrorDialog(dialogService);
         }
 
         if (this.isConnectedToDocument()) {
+            initializeCitationAdapter(this.getXTextDocument().get());
             dialogService.notify(Localization.lang("Connected to document") + ": "
                     + this.getCurrentDocumentTitle().orElse(""));
+            try {
+                this.cslCitationOOAdapter = new CSLCitationOOAdapter(this.getXTextDocument().get());
```

Comment:
duplicated code to  initializeCitationAdapter?

Reply:
@subhramit  Check again does the same as

Reply:
resolved